packages/mobile/src/web3/saga.ts

```diff
@@ -36,68 +34,67 @@ const MNEMONIC_BIT_LENGTH = (ETH_PRIVATE_KEY_LENGTH * 8) / 2
 
 const TAG = 'web3/saga'
 // The timeout for web3 to complete syncing and the latestBlock to be > 0
-const CHECK_SYNC_PROGRESS_TIMEOUT = 60000
+export const SYNC_TIMEOUT = 60000
```

Comment:
Renamed this mostly because of the same VScode import ordering bug mentioned above